.flake8

```
@@ -1,4 +1,7 @@
 [flake8]
+# Don't even try to analyze these:
+extend-exclude =
+    blog/migrations/*
```

Is the wildcard really needed here? Just listing the dirs should exclude everything recursively:
```suggestion
  blog/migrations/,
```

---

After this commit I got this error:
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

blog/migrations/0001_initial.py:36:80: E501 line too long (80 > 79 characters)
blog/migrations/__init__.py:1:80: E501 line too long (85 > 79 characters)
```

---

I changed again to `blog/migrations/*`